wiring/src/spark_wiring_fuel.cpp

```diff
@@ -108,7 +108,7 @@ float FuelGauge::getVCell() {
 	byte MSB = 0;
 	byte LSB = 0;
 
-	readRegister(VCELL_REGISTER, MSB, LSB);
+	CHECK(readRegister(VCELL_REGISTER, MSB, LSB));
```

Comment:
Returning numeric errors codes through a method that normally returns a float seems strange. IMO we should return some fixed documented value not overlapping with normal range on errors.

Reply:
Sounds good. 👍